app/containers/EmojiPicker/EmojiCategory.tsx

```diff
@@ -1,43 +1,41 @@
 import React from 'react';
-import { useWindowDimensions } from 'react-native';
 import { FlatList } from 'react-native-gesture-handler';
 
-import { EMOJI_BUTTON_SIZE } from './styles';
-import scrollPersistTaps from '../../lib/methods/helpers/scrollPersistTaps';
 import { IEmoji } from '../../definitions/IEmoji';
+import scrollPersistTaps from '../../lib/methods/helpers/scrollPersistTaps';
 import { PressableEmoji } from './PressableEmoji';
+import { EMOJI_BUTTON_SIZE } from './styles';
 
 interface IEmojiCategoryProps {
 	emojis: IEmoji[];
 	onEmojiSelected: (emoji: IEmoji) => void;
 	tabLabel?: string; // needed for react-native-scrollable-tab-view only
+	windowWidth: number;
 }
 
-const EmojiCategory = ({ onEmojiSelected, emojis }: IEmojiCategoryProps): React.ReactElement | null => {
-	const { width } = useWindowDimensions();
-
-	const numColumns = Math.trunc(width / EMOJI_BUTTON_SIZE);
-	const marginHorizontal = (width % EMOJI_BUTTON_SIZE) / 2;
+const EmojiCategory = ({ onEmojiSelected, emojis, windowWidth }: IEmojiCategoryProps): React.ReactElement | null => {
+	const numColumns = windowWidth / EMOJI_BUTTON_SIZE;
+	const canBeRound = numColumns % 1 > 0.75; // pixel logic
```

Comment:
Can you elaborate on this logic?
Perhaps some comments for future us?
My point is the previous logic was quite simple: math to get number os columns and horizontal margin.

Where did you get 0.75 from?
What's going to happen in case of a OS wise change on a11y?